api/collections/serializers.py

```diff
@@ -24,6 +25,7 @@ class CollectionSerializer(JSONAPISerializer):
     title = ser.CharField(required=True)
     date_created = ser.DateTimeField(read_only=True)
     date_modified = ser.DateTimeField(read_only=True)
+    bookmarks = ser.BooleanField(read_only=False, default=False, source='is_bookmark_collection')
```

Comment:
"bookmarks" implies that the field is a list. Might make sense to rename this to `bookmark`.